api_gen.py

```diff
@@ -91,10 +91,11 @@ def __init__(self, text):
         self.fname = parts['fname']
         self.sig = parts['sig']
 
-        self.args = self.SIG_PATTERN.findall(self.sig)
+        sig_const_stripped = self.sig.replace('const', '')
+        self.args = self.SIG_PATTERN.findall(sig_const_stripped)
         if self.args is None:
```

Comment:
`self.args` will only be `None` if the regex fails to match, if it is non-None, then it found one or more matches.